src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/mobile-sdk-api-guide.mdx

```diff
@@ -0,0 +1,363 @@
+          * [Record breadcrumb events](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb)
+          * [Set the maximum size of an event pool](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-max-event-pool-size)
+          * [Set maximum time agent stores events in memory](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-max-event-buffer-time)
+          * [Return current session ID](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/current-session-id)
+          * [Set a custom user ID to associate with events and attributes](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-custom-user-id)
+
+        For more about which would be the best method to use and why, see [Add custom data](/docs/mobile-monitoring/new-relic-mobile/maintenance/add-custom-data-new-relic-mobile).
+      </td>
+    </tr>
+
+    <tr>
+      <td id="track-custom">
+        Track custom network requests and failures.
+      </td>
+
+      <td>
+        [Track HTTP requests](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/network-request-success)
+
+        [Track failing HTTP requests](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/network-request-failures)
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Record a handled exception as an event, including the context of what happened.
+      </td>
+
+      <td>
+        [Record handled exceptions](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-handled-exceptions)
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Record errors.
+      </td>
+
+      <td>
+        [Record errors](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-errors)
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Name activity for app launch time.
+      </td>
+
+      <td>
+        [Name activity for app launch time](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/name-android-activity)
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Shut down the agent.
+      </td>
+
+      <td>
+        [Shut down the agent](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/shut-down-agent).
+      </td>
+    </tr>
+
+        <tr>
+      <td>
+        Enable/disable default mobile monitoring settings.
+      </td>
+
+      <td>
+        [Enable/disable monitoring features](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings)
+      </td>
+    </tr>
+
+            <tr>
+      <td>
+        Configure cross-app tracing to link transactions between your mobile and APM-monitored applications.
+      </td>
+
+      <td>
+        [Configure cross-app tracing](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/cross-app-tracing)
+      </td>
+    </tr>
+
+                <tr>
+      <td>
+        Run a test crash report.
+      </td>
+
+      <td>
+        [Test crash reporting](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/test-crash-reporting)
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## Android usage guide [#android-sdk-usage]
```

Comment:
Note to self - build these sections out with SMEs, or rename the section in the meantime and create a followup ticket